<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_habit_add</class>
 <widget class="QDialog" name="dialog_habit_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">SmoothScrollArea{
	background: transparent; 
	border: none;
}

#scrollAreaWidgetContents{
	border-radius: 6px;
	background-color: rgb(249, 249, 249);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="StrongBodyLabel" name="label_10">
       <property name="text">
        <string>图标</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="SmoothScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>458</width>
          <height>98</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="BodyLabel" name="label_9">
       <property name="text">
        <string>（小于64字）</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="StrongBodyLabel" name="label">
       <property name="text">
        <string>习惯内容</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="LineEdit" name="lineEdit_content">
       <property name="placeholderText">
        <string>示例：早睡早起</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="BodyLabel" name="label_3">
       <property name="text">
        <string>是否开启随机浮动</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="SwitchButton" name="switchButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="BodyLabel" name="label_2">
       <property name="text">
        <string>获得奖杯值</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="StrongBodyLabel" name="label_4">
       <property name="text">
        <string>奖励选项</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="LineEdit" name="lineEdit_reward">
       <property name="placeholderText">
        <string>示例：666</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="2">
      <widget class="BodyLabel" name="label_8">
       <property name="text">
        <string>可完成几次</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="SpinBox" name="spinBox_times">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="StrongBodyLabel" name="label_5">
       <property name="text">
        <string>周期选项</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="BodyLabel" name="label_7">
       <property name="text">
        <string>每几天</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="BodyLabel" name="label_6">
       <property name="text">
        <string>预设</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="SpinBox" name="spinBox_days">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="ComboBox" name="comboBox">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
